draft-ietf-quic-transport.md

```diff
@@ -2496,9 +2478,8 @@ ensure connection stability. This section describes the protocol for migrating a
 connection to a preferred server address.
 
 Migrating a connection to a new server address mid-connection is left for future
-work. If a client receives packets from a new server address not indicated by
-the preferred_address transport parameter, the client SHOULD discard these
-packets.
+work. If a client receives packets from a new server address when the client has
+not initiated a migration, the client SHOULD discard these packets.
```

Comment:
This is slightly different.  It's not as tight as it was, because it could be read to allow any server address to be used if the client decides to migrate.  That's probably not your intent.